src/crate/client/doctests/client.txt

```diff
@@ -179,7 +179,7 @@ supported, all other fields are 'None'::
     >>> result = cursor.fetchone()
     >>> pprint(result)
     ['Aldebaran',
-     1373932800000,
+     datetime.datetime(2013, 7, 15, 18, 0),
```

Comment:
This currently raises an exception on CI, see [1]. The current code expects `datetime.datetime(2013, 7, 16, 0, 0)` here.

[1] https://github.com/crate/crate-python/runs/2114206441?check_suite_focus=true#step:5:388